anvil_consortium_manager/templates/anvil_consortium_manager/managedgroup_detail.html

```diff
@@ -73,6 +73,22 @@ <h2 class="accordion-header" id="headingWorkspaceAccessOne">
   <div class="my-3">
     <div class="accordion" id="accordionMembers">
 
+      <div class="accordion-item">
+        <h2 class="accordion-header" id="headingParents">
+          <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapseParents" aria-expanded="false" aria-controls="collapseParents">
+            <span class="fa-solid fa-code-fork mx-2" data-fa-transform="flip-v"></span>
+            View group Parents
```

Comment:
Let's change this to "View groups that this group is in"

Reply:
The change is made.